squash/dashboard/templates/dashboard/metric.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'dashboard/dashboard.html' %}
+
+{% block dashboard %}
+
+   <div class="container">
+      <div class="row">
+        <div class="page-header">
+          <h1>KPM CI<small> Tracking the impact of code changes in the Key Performance Metrics </small></h1>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-xs-2">
+          <ul class="nav nav-pills nav-stacked">
+
+              <!-- using fixed metric list for now -->
+
+              <li role="presentation"><h4>Metrics</h4></li>
+              <hr>
+              <li role="presentation">Astrometry</li>
+              <li role="presentation"><a href="">AM1</a></li>
+              <li role="presentation"><a href="">AM2</a></li>
+              <li role="presentation">Photometry Repeatability</li>
+              <li role="presentation"><a href="">PA1</a></li>
+              <li role="presentation"><a href="">PA2</a></li>
+           </ul>
+        </div>
+        <div class="col-xs-10">
+            <!-- Using a fixed dataset for now, later this could be a pull down menu for selecting different datasets -->
+
+            <h4 class="pull-right">
+                <div class="dropdown">
+                    <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown">ci_hsc
+                        <span class="caret"></span></button>
+                    <ul class="dropdown-menu">
+                        <li><a href="#">ci_hsc</a></li>
+                        <li><a href="#">ci_cfht</a></li>
```

Comment:
I noticed this dropdown doesn't actually drop down to show the `ci_cfht` option.